### ISSUE TITLE
Incompatibility with ObjectValue

### ISSUE THREAD

**Issue by medallyon** [rojo-rbx/rojo#704]
I have an ObjectValue in my Workspace pointing to a Folder in ReplicatedStorage, but when building the place, the Value referencing the Folder is empty. I tried changing the load order in my default.project.json to put ReplicatedStorage first, but it doesn't look like that has any effect.

**Comment by tachy0n-z** [comment]
Yeah Rojo isn't too cooperative with `ObjectValue` instances. You might just have to find a workaround or reference it directly.
